File: 1.TwoSum/Two_Sum.c
int* twoSum(int* nums, int numsSize, int target, int* returnSize) {
    
    for(int i = 0; i < numsSize; i++)
    {
        for(int j = 0; j < numsSize; j++)
        {
            if((i != j )&&( nums[i]+nums[j] == target))
            {
                int* result = (int*)malloc(2*sizeof(int));
                result[0] = i;
                result[1] = j;
                *returnSize = 2;
                return result;
            }
        }
    }
    return 0;
}

File: 0-99/1-9/1.TwoSum/Two_Sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        dict = {}
        idx = 0
        for x in nums:
            if x in dict.keys():
                dict[x].append(idx)
            else:
                dict[x] = [nums.index(x)]
            idx += 1
        for x in dict.keys():
            y = target - x
            if y in dict.keys():
                if y != x:
                    return [nums.index(x), nums.index(y)]
                else:
                    try:
                        list = dict[x]
                        if len(list) == 2:
                            return list
                    except:
                        pass